CMakeLists.txt

```diff
@@ -16,15 +16,15 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/build/private/local_config.cmake)
   include(${CMAKE_CURRENT_SOURCE_DIR}/build/private/local_config.cmake)
 endif()
 
+
```

Comment:
As we already agreed, please do a diff before sending the PR. Remove unnecessary blank lines in PR please.